docs/api-docs/cart-and-checkout/cart-and-checkout-overview.mdx

```diff
@@ -129,15 +129,19 @@ We will go over common Storefront Cart and Checkout as well as Server-to-Server
 ### Please create some text for the API option [422]
 *Server to Server Cart API*
 
-***Issue:*** When a cart contains a product that has an incorrect or missing text modifier.
+***Issue:*** When a cart contains a product with an incorrect or missing text modifier.
 
 ***Resolution:***
-Options and modifiers refer to a list of choices on a product. Options used to build out variants and modifiers are not tied to variants at all. To learn more about options and modifiers, see [Products Overview](/api-docs/catalog/products-overview#products-overview_modifier-options).
+Options and modifiers refer to a list of choices on a product. Options used to build out variants and modifiers are not tied to variants. To learn more about options and modifiers, see [Products Overview](/api-docs/catalog/products-overview#products-overview_modifier-options).
 
-To add a product to the cart with a single modifier (text field), POST to the [Cart API](/docs/rest-management/carts/carts-single#create-a-cart) without the `variant_id`.
+To update a cart line item to fix an incorrect or missing text modifier, use the [Update cart line item](https://developer.bigcommerce.com/docs/rest-management/carts/items#update-cart-line-item) endpoint. 
 
+```http filename="Example cart line item request" showLineNumbers copy
+PUT https://api.bigcommerce.com/stores/{store_hash}/v3/carts/{cartId}/items/{itemId}
+x-auth-token: {{access_token}}
+accept: application/json
+content-type: application/json
 
-```json filename="Single modifier" showLineNumbers copy
 {
```

Comment:
is this the correct request body for this endpoint?

Reply:
It matches the request body[ here](https://developer.bigcommerce.com/docs/rest-management/carts/items#update-cart-line-item) when you select `Item with modifier`.